pkg/machines/constants/store-action-types.es6

```diff
@@ -33,7 +33,7 @@ export const UPDATE_ADD_VM = "UPDATE_ADD_VM";
 export const UPDATE_ADD_STORAGE_POOL = "UPDATE_ADD_STORAGE_POOL";
 export const UPDATE_LIBVIRT_STATE = "UPDATE_LIBVIRT_STATE";
 export const UPDATE_OS_INFO_LIST = "UPDATE_OS_INFO_LIST";
-export const UPDATE_STORAGE_VOLUMES = "UPDATE STORAGE_VOLUMES";
+export const UPDATE_STORAGE_VOLUMES = "UPDATE_STORAGE_VOLUMES";
```

Comment:
This typo was not affecting anything because we use the const ENUM and **not** the string itself all around the code. The code for storage volumes is tested :)